packages/editor/src/store/selectors.js

```diff
@@ -1081,10 +1071,11 @@ export function getActivePostLock( state ) {
  * @return {boolean} Whether the user can or can't post unfiltered HTML.
  */
 export function canUserUseUnfilteredHTML( state ) {
-	return has( getCurrentPost( state ), [
-		'_links',
-		'wp:action-unfiltered-html',
-	] );
+	return (
+		getCurrentPost( state )._links?.hasOwnProperty(
+			'wp:action-unfiltered-html'
+		) ?? false
+	);
```

Comment:
Ditto here about using `Boolean()`

Reply:
Right 👍 Addressed in a91d22711b530262ce14f2589846b26647fb8db7